src/oom/oomd.c

```diff
@@ -21,13 +21,13 @@
 static bool arg_dry_run = false;
 static int arg_swap_used_limit_permyriad = -1;
 static int arg_mem_pressure_limit_permyriad = -1;
-static usec_t arg_mem_pressure_usec = 0;
+static usec_t arg_mem_pressure_usec = USEC_INFINITY;
 
 static int parse_config(void) {
         static const ConfigTableItem items[] = {
                 { "OOM", "SwapUsedLimit",                    config_parse_permyriad, 0, &arg_swap_used_limit_permyriad    },
                 { "OOM", "DefaultMemoryPressureLimit",       config_parse_permyriad, 0, &arg_mem_pressure_limit_permyriad },
-                { "OOM", "DefaultMemoryPressureDurationSec", config_parse_sec,       0, &arg_mem_pressure_usec            },
+                { "OOM", "DefaultMemoryPressureDurationSec", config_parse_sec_fix_0, 0, &arg_mem_pressure_usec            },
```

Comment:
I think we should introduce specific parser `config_parse_duration()` or so. Then, do mostly equivalent to the conf parser for PID1, but maps an empty string and 0 to the default duration.

Reply:
Then, please add a short comment that mapping from 0 to default is for backward compat.

Reply:
Note I also allow setting "infinity" here since technically it was allowed previously. Not sure if we want to disallow it now...